lib/money/money.rb

```diff
@@ -153,7 +155,12 @@ def -@
   end
 
   def <=>(other)
-    return unless other.respond_to?(:to_money)
+    if other.is_a?(Numeric)
```

Comment:
this will also speed up significantly comparing with a Numeric by not casting these to money objects